released/packages/coq-fourcolor-reals/coq-fourcolor-reals.1.4.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+
+homepage: "https://github.com/coq-community/fourcolor"
+dev-repo: "git+https://github.com/coq-community/fourcolor.git"
+bug-reports: "https://github.com/coq-community/fourcolor/issues"
+license: "CECILL-B"
+
+synopsis: "Interface for real numbers used in the Four Color Theorem"
+description: """
+An axiomatization of the setoid of classical real numbers, along
+with proofs of properties such as categoricity."""
+
+build: [make "-C" "theories/reals" "-j%{jobs}%"]
+install: [make "-C" "theories/reals" "install"]
+depends: [
+  ("coq" {>= "8.16" & < "8.21~"} |
+   "rocq-core"{ (>= "9.0" & < "9.1~") | (= "dev")0})
```

Comment:
Same here. Also, what is that spurious zero in the formula for `rocq-core`? Was it tested?